packages/jbrowse-plugin-apollo/src/components/CreateApolloAnnotation.tsx

```diff
@@ -226,7 +315,7 @@ export function CreateApolloAnnotation({
                   onChange={handleParentFeatureCheck}
                 />
               }
-              label={`${annotationFeature.type}:${annotationFeature.min}..${annotationFeature.max}`}
+              label={`${getFeatureNameOrId(annotationFeature, apolloSessionModel)} (${annotationFeature.min}..${annotationFeature.max})`}
```

Comment:
Suggestion: Use `{annotationFeature.min + 1}` and on line 337 `{child.min + 1}` so the coordinate system is 1-based and consistent with the UI.

![image](https://github.com/user-attachments/assets/22df10ce-4b78-4c1d-8570-f1ae696e4a60)